package.json

```diff
@@ -13,6 +13,8 @@
     "test": "echo \"There are no tests :(\"",
     "build": "ncc build -o dist index.js",
     "format": "prettier --write index.js src/ping.js",
+    "format:markdown": "prettier --write \"**/*.md\"",
+    "format:markdown:check": "prettier --check \"**/*.md\"",
     "lint": "eslint",
     "check:markdown": "find *.md docs/*.md -print0 | xargs -0 -n1 markdown-link-check",
```

Comment:
A bit of a nit, but these read a bit backwards when put together, where it seems unintuitive if I tried to guess the commands and what they did. 'check:markdown' vs 'format:markdown:check'. Maybe we should update the 'check:markdown' to be 'check:markdown:links'? I dunno, what do you think?

Reply:
Thanks for your suggestion! I've used `check:markdown-links` with `check` as verb and object `markdown-links`. I think that makes the meaning of the script name more intuitively readable.